pyccel/parser/semantic.py

```diff
@@ -1966,9 +1969,13 @@ def _assign_GeneratorComprehension(self, lhs_name, expr):
         elif isinstance(expr, FunctionalMin):
             val = math_constants['inf']
             d_var = self._infer_type(result)
+            if d_var['class_type'] is PythonNativeInt():
+                val = LiteralInteger(2**63-1)
```

Comment:
Did you create an issue about these values?

Reply:
fixed